src/components/MentorDashbord/common/MentorDashboardSidebarStyled.js

```diff
@@ -1,33 +1,91 @@
 import styled from 'styled-components';
 
+const active = `    background: #004699;
```

Comment:
Codacy found an issue: [The numeric literal '004699' will have at different value at runtime.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5859548)